packages/calcite-components/src/components/segmented-control/segmented-control.e2e.ts

```diff
@@ -193,6 +193,33 @@ describe("calcite-segmented-control", () => {
     expect(await getSelectedItemValue(page)).toBe("2");
   });
 
+  it("updates selection when cleared with undefined", async () => {
+    async function getSelectedItemValue(page: E2EPage): Promise<string> {
```

Comment:
Can you move `getSelectedItemValue` outside the `it` block and reuse between this and the above test?